### ISSUE TITLE
Updating plugin

### ISSUE THREAD

**Issue by yaitsbilly** [r-clancy/PlugMan#85]
Can I use this plugin to unload a plugin to update then load it? Trying to use to to update Mypets. I did unload, move new jar in and old jar out, did load and its giving me and error while enabling plugin.

**Comment by kennyrkun** [comment]
Yes, however some plugins don't support reloading in this manner (for example EssentialsX will throw a fit if you reload it).

I've use this plugin constantly to reload my plugins while I'm developing them and have never had any problems with it.
